chapters/functions.tex

```diff
@@ -284,6 +284,12 @@ \section{Pure Modelica Functions}\label{pure-modelica-functions}
 The reason is that solving systems of equations generally requires expressions to be evaluated an unknown number of times.
 This includes the special handling of \lstinline!when initial()! during initialization.
 
+It is still possible to call an impure function in a system of equations in specific ways.
+Specifically for the deprecated case of external functions assumed to be impure, and when using \lstinline!pure(expression)! to call an impure function in a pure function.
```

Comment:
Looks good technically, but could be written in a slightly more readable fashion.  For example, it would be nice to reduce the number of _specific…_ in the section.
```suggestion
There are two ways in which an impure function could be called in a system of equations,
namely in the deprecated case of external functions assumed to be impure, and when using \lstinline!pure($\ldots$)! to call an impure function from within a pure function.
```
(The alternatve to `\ldots` above would be `\mathit{expression}`.)